app/components/header_component.html.erb

```diff
@@ -15,7 +15,7 @@
       <span class="menu-button__text">Menu</span>
     <% end %>
   </div>
-  <%= render Header::NavigationComponent.new(extra_resources: { "/events" => "Events", "/blog" => "Blog" }) %>
```

Comment:
@EllieNodder I don't think we can take this out of the nav just yet - not until the home page is redesigned as we have a clearer signpost to Events on there.